zerver/tests/test_push_notifications.py

```diff
@@ -4361,6 +4373,7 @@ def test_get_message_payload_gcm_direct_message(self) -> None:
                 "server": settings.EXTERNAL_HOST,
                 "realm_id": hamlet.realm.id,
                 "realm_name": hamlet.realm.name,
+                "realm_url": hamlet.realm.url,
                 "realm_uri": hamlet.realm.url,
```

Comment:
`l` is after `i` in the alphabet, so there are all alphabetized incorrectly.

Reply:
Fixed.